### PR TITLE
DENG-4446 add profile_group_id to clients_daily_v6, clients_daily_joined_v1, clients_first_seen_v1, clients_last_seen_v1, and clients_last_seen_v2

### PR DESCRIPTION
**Pull request by kwindau**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-4477)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql [hunk 1]
```diff
@@ -1573,7 +1575,7 @@ udf_aggregates AS (
     aggregates
 )
 SELECT
-  * EXCEPT (map_sum_aggregates),
+  * EXCEPT (profile_group_id, map_sum_aggregates),
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql:1578]: I think you don't need to explicitly exclude and add `profile_id_group` at the end here. BigQuery will add this column automatically to the end of the existing table schema

Comment (kwindau) [reply sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql]: Good to know, thanks! Updated the query, please re-review when you get time

**Review by Marlene-M-Hirose — APPROVED** [review]

**Review by scholtzan — APPROVED** [review]
Definitely worth checking whether everything looks good on triage tomorrow.

**Comment by kwindau** [comment]
> Definitely worth checking whether everything looks good on triage tomorrow.

Praying for the best and going to keep a close eye on this one, it's definitely risky... fingers crossed

**Pull request merged**